htdocs/societe/card.php

```diff
@@ -1291,8 +1291,10 @@ function init_check_no_email(input) {
 			print '<span id="TypeName" class="fieldrequired">'.$form->editfieldkey('ThirdPartyName', 'name', '', $object, 0).'</span>';
 		}
 		print '</td><td'.(empty($conf->global->SOCIETE_USEPREFIX) ? ' colspan="3"' : '').'>';
-		print '<input type="text" class="minwidth300" maxlength="128" name="name" id="name" value="'.dol_escape_htmltag($object->name).'" autofocus="autofocus">';
-		print $form->widgetForTranslation("name", $object, $permissiontoadd, 'string', 'alpahnohtml', 'minwidth300');
+		//print '<input type="text" class="minwidth300" maxlength="128" name="name" id="name" value="'.dol_escape_htmltag($object->name).'" autofocus="autofocus">';
```

Comment:
I would prefer a more powerful implementation that will be an ajax check when we type the name of a company. After 3 chars entered, a javascript ajax call try to find some similar thirdparties and if found, show a warning popup "Warning: we found x thirdparties that looks similar: xxx, yyy, ..."
So we will be able to enhance the engine for "duplicate search" with more powerful rules (case insensitive, check even if typo, check ingoring accents, ...) than a simple combo with the rule "contains stubstring".